.github/workflows/release.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: npm run build
 
       - name: Release
-        run: npm run release:ci -- ${{github.event.inputs.releaseType}}
+        run: npm run release:ci -- ${{github.event.inputs.releaseType}} --preRelease=${{github.event.inputs.preReleaseFlavor}}
```

Comment:
There won't be any issues if `releaseType` is set and `--preRelease=` is empty?

Reply:
Let me add some checks for it